rpi_backlight/__init__.py

```diff
@@ -27,10 +37,21 @@ class Backlight:
     """Main class to access and control the display backlight power and brightness."""
 
     def __init__(
-        self, backlight_sysfs_path: Union[str, "PathLike[str]"] = _BACKLIGHT_SYSFS_PATH
+        self,
+        backlight_sysfs_path: Optional[Union[str, "PathLike[str]"]] = None,
+        board_type: BoardType = BoardType.RASPBERRY_PI,
```

Comment:
Maybe we should swap these two so `board_type` comes first and can be passed in as a positional argument easily - what do you think?

Reply:
I played around with this early- I am okay to swap!